ais/api/serializers.py

```diff
@@ -52,9 +52,10 @@ def render(self, data):
 class AddressJsonSerializer (GeoJSONSerializer):
     excluded_tags = ('info_resident', 'info_company', 'voter_name')
 
-    def __init__(self, geom_type='centroid', geom_source=None, **kwargs):
+    def __init__(self, geom_type='centroid', geom_source=None, in_street=False, **kwargs):
         self.geom_type = geom_type
         self.geom_source = geom_source
+        self.in_street = in_street
```

Comment:
When i do this i get: TypeError: __init__() got an unexpected keyword argument 'in_street'

fyi, i just pushed changes incorporating all your comments except for this one